grails-app/domain/imod/LearningObjective.groovy

```diff
@@ -64,9 +64,7 @@ class LearningObjective {
 	]
 
 	public static final List genericConditions = [
-		'Given a problem specification',

```

Comment:
I thought we were keeping Generic Conditions, but not allowing them to cross over into Custom Conditions.
Was there a decision to remove Generic Conditions?
@nishantk12 @vidyaks